### ISSUE TITLE
 [Vue warn]: Duplicate presense of slot "default" found in the same render tree - this will likely cause render errors.

### ISSUE THREAD

**Issue by myronliu347** [vuejs/vue#3676]
use  slot but  Has been an error

```html
<!--touchRipple-->
  <div @mousedown="handleMouseDown" @mouseup="end()" @mouseleave="end()" @touchstart="handleTouchStart"  @touchend="end()" @touchcancel="end()">
    <div :style="style">
      <circle-ripple :key="ripple.key" v-for="ripple in ripples"></circle-ripple>
    </div>
    <slot></slot>
  </div>
```

```html
  <touch-ripple>
      <icon :value="icon" :class="[iconClass]"></icon>
   </touch-ripple>
```

**Comment by LinusBorg** [comment]
Which version? This has been fixed in [`2.0.0-rc.6`](https://github.com/vuejs/vue/releases/tag/v2.0.0-rc.6) (issue #3642)

**Comment by myronliu347** [comment]
Yes is ok,  Thank you !
